roles/matrix-bot-maubot/templates/systemd/matrix-bot-maubot.service.j2

```diff
@@ -0,0 +1,42 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Maubot
+{% for service in matrix_bot_maubot_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_bot_maubot_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ matrix_systemd_unit_home_path }}"
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} kill matrix-bot-maubot 2>/dev/null || true'
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} rm matrix-bot-maubot 2>/dev/null || true'
+
+ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-bot-maubot \
+            --log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+            --read-only \
+            --cap-drop=ALL \
```

Comment:
Looks like indentation is mixed here. Some lines are using spaces, some are using tabs.

Better switch everything to tabs in this file.

Reply:
done